<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>871</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Directory to Scan</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtDirToScan">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPickDirToScan">
       <property name="text">
        <string>Pick Directory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStartScan">
       <property name="text">
        <string>Start Scan</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lblPic1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>261</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPrev1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>550</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNext1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>550</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFileName1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>File Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblInfo1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>510</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Capture Date</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCopy1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>590</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPic3">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>200</y>
      <width>261</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPrev3">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>550</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblInfo3">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>510</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Capture Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFileName3">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>470</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>File Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCopy3">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>590</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNext3">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>550</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPic2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>200</y>
      <width>261</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblFileName2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>470</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>File Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>580</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="selectedDateTimeEdit2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>510</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>MM/dd/yyyy hh:mm:ss AP</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelete2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>580</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
